app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt

```diff
@@ -5,6 +5,7 @@ import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.LinearSnapHelper
 import androidx.recyclerview.widget.OrientationHelper
 import androidx.recyclerview.widget.RecyclerView
+import kotlin.Throws
```

Comment:
Why is this needed? It doesn't seem to be used. Suggest removing it.

Reply:
@BenHenning Actually, I mentioned this in the issue. 
It is used in the below line not sure if this is needed 
https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt#L18

Reply:
@riturajjain2000 The PR reviewer can resolve the conversation after confirming the suggested changes are done or if anything left to discuss. The conversation should not be resolved by PR authors.

Reply:
Ok ,I'll keep that in mind